src/src/components/Molecules/Portal.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
Not a big deal, but I feel like it can be helpful for readability l to put React methods directly here instead of throughout the file using React.useEffect() etc.

Reply:
Interesting, can you explain more why you think so?

I prefer `React.useEffect()` for readability as it makes it clear in code that the hook is comming from react, as opposed to a user hook.

Basically, using the `React.` namespace makes it easy to differentiate default react hooks from my custom hooks.